docs/nuxt.config.js

```diff
@@ -1,3 +1,7 @@
 import theme from '@nuxt/content-theme-docs'
 
-export default theme({})
+export default theme({
+  docs: {
+    primaryColor: '#564995'
```

Comment:
This color is very bad in terms of contrast when using the dark theme.

![Screenshot 2020-12-14 at 23 23 21](https://user-images.githubusercontent.com/153197/102143029-9e182900-3e63-11eb-9ec5-d44ebb1cc059.png)

(The contrast can be tested with the Lighthouse developer tool, using accessibility checks)

Reply:
Thanks for the tip about "Lighthouse". i never used it, very cool to discover it :) 
I will improve the primary color so 👍

Reply:
Is it possible to define different colors depending on dark/light theme? I'm asking because with latest changes now the light theme is not contrasting enough (not that some of those are issues from Nuxt theme itself but all the purple links also have a contrast problem):

![Screenshot 2020-12-16 at 09 17 03](https://user-images.githubusercontent.com/153197/102322655-ac5b6780-3f7f-11eb-9fa8-a3226645e9f3.png)

Reply:
Currently no unfortunatly. We can only define one color for both ... 
Later with new docs theme, we we'll be able to 😀 
Can you purpose a doc color to solve this contrast for light & dark theme around the logo color ?

Reply:
> Can you purpose a doc color to solve this contrast for light & dark theme around the logo color ?

That's not possible, I think. We need to be able to set different colors for dark/light mode.